mlflow/tracking/fluent.py

```diff
@@ -256,9 +257,13 @@ def log_artifact(local_path, artifact_path=None):
     :param artifact_path: If provided, the directory in ``artifact_uri`` to write to.
     """
     run_id = _get_or_start_run().info.run_id
+    if os.path.isdir(local_path):
+        MlflowClient().log_artifacts(run_id, local_path, artifact_path)
```

Comment:
Should probably put the `log_artifact` call below in an `else` :P

Reply:
Or actually can't we just always call `MlflowClient.log_artifact()` now that it's been fixed to accept a file or directory?

Reply:
Good call, thanks